data/sql/updates/pending_db_world/rev_1548778743987887239.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1548778743987887239');
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 32711;
+
+DELETE FROM `creature` WHERE `guid` = 2023271;
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`)
+VALUES
+(2023271,32751,571,0,0,1,1,28212,0,5808.4,583.112,652.386,5.0091,300,0,0,10635,0,0,0,0,0,'',0);
+
+DELETE FROM `creature_formations` WHERE `leaderGUID` = 114764;
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`)
+VALUES
+(114764,114764,0,0,515,0,0),
+(114764,2023271,2,270,515,0,0);
+
+DELETE FROM `creature_text` WHERE `CreatureID` IN (27047,32711,29506);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`)
+VALUES
+(27047, 0, 0, '', 12, 0, -1, 0, 0, 0, 0, 0, 'Invisible Stalker (Floating Only)'),
```

Comment:
Probability is an unsigned column and can't be set to -1.

Reply:
From commit b34bc28e5b02514fca3519beac420c58faa89cad:
```
ALTER TABLE `creature_text` CHANGE `probability` `Probability` FLOAT UNSIGNED NOT NULL DEFAULT '0';
```

Prior to the above commit, the column was signed.

Reply:
@Stoabrogga were you using latest AC version when you made this fix?

Reply:
Thanks for pointing this out, I did indeed miss the warning. As mentioned in the PR, I'm using commit 9af2a461f65316ce4c21c9668dd62515b8e4f370. Pushed a new commit to fix this.